packages/components/Avatar/src/Avatar.tsx

```diff
@@ -53,10 +54,17 @@ export const Avatar = compose<AvatarType>({
       const { badge, ...mergedProps } = avatar.props;
       const svgIconsEnabled = ['ios', 'macos', 'win32', 'android'].includes(Platform.OS as string);
       const RingComponent = showRing && !transparentRing ? Slots.ring : Fragment;
+      const hasOuterRing = Platform.OS === 'android';
```

Comment:
Does android always have an outer ring?

Reply:
Yes, in case of presence active - there always is an outer ring for android.